### PR TITLE
Create fortios.markdown

### PR DESCRIPTION
**Pull request by kimfrellsen**
this is documentation for the initial version of a new device_tracker component named Fortios from Fortinet.

**Description:**
the fortios device_tracker.py pull request is not created yet. But I'll make a PR during this week and documentation should be ready when the code is committed :)

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/device_tracker.fortios.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
```
Comment (frenck) [inline source/_components/device_tracker.fortios.markdown]: The documentation header is missing, please add it.

File: source/_components/device_tracker.fortios.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+This component enables Home Assistant to do device tracking of devices with a MAC address connected to a FortiGate from [Fortinet](https://www.fortinet.com).
+
+The component rely on fortiosapi: https://pypi.org/project/fortiosapi/
```
Comment (frenck) [inline source/_components/device_tracker.fortios.markdown]:
```suggestion
The component relies on the [fortiosapi](https://pypi.org/project/fortiosapi/).
```

File: source/_components/device_tracker.fortios.markdown [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+The component has been tested both on FortiGate appliance and FortiGate VM running SW FortiOS v. 6.0.x and 6.2.0.
+
+All devices with a MAC address identified by FortiGate would be tracked, this covers both Ethernet and WiFi devices, incl. devices detected by LLDP.
```
Comment (frenck) [inline source/_components/device_tracker.fortios.markdown]:
```suggestion
All devices with a MAC address identified by FortiGate would be tracked, this covers both Ethernet and WiFi devices, including devices detected by LLDP.
```

File: source/_components/device_tracker.fortios.markdown [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+device_tracker:
+  - platform: fortios
+    host: 192.168.0.1
+    token: YOUR_API_USER_KEY
+    verify_ssl: False
+```
+
+{% configuration %}
+host:
+    description: Hostname or IP address of the FortiGate
```
Comment (frenck) [inline source/_components/device_tracker.fortios.markdown]:
```suggestion
    description: Hostname or IP address of the FortiGate.
```

File: source/_components/device_tracker.fortios.markdown [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+All devices with a MAC address identified by FortiGate would be tracked, this covers both Ethernet and WiFi devices, incl. devices detected by LLDP.
+
+The component is based on the Home Assistant `device_tracker` platform.
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: fortios
+    host: 192.168.0.1
```
Comment (frenck) [inline source/_components/device_tracker.fortios.markdown]:
```suggestion
    host: YOUR_HOST
```

File: source/_components/device_tracker.fortios.markdown [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    token: YOUR_API_USER_KEY
+    verify_ssl: False
```
Comment (frenck) [inline source/_components/device_tracker.fortios.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/device_tracker.fortios.markdown [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+host:
+    description: Hostname or IP address of the FortiGate
+    required: true
+    type: string
+token:
+    description: See [Fortinet Devloper Network](https://fndn.fortinet.com) for how to create a API token. Remember this component only needs read access to a FortiGate, so configure the API user to only to have limited and read-only access.
+    required: true
+    type: string
+  verify_ssl:
+    description: if the SSL certificate should be verified. In most home cases users do not have a verified certificate. Default=False.
```
Comment (frenck) [inline source/_components/device_tracker.fortios.markdown]:
```suggestion
    description: if the SSL certificate should be verified. In most home cases users do not have a verified certificate.
```

The default should not be in the description but added to the configuration list. Please add it.

File: source/_components/device_tracker.fortios.markdown [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    verify_ssl: False
+```
+
+{% configuration %}
+host:
+    description: Hostname or IP address of the FortiGate
+    required: true
+    type: string
+token:
+    description: See [Fortinet Devloper Network](https://fndn.fortinet.com) for how to create a API token. Remember this component only needs read access to a FortiGate, so configure the API user to only to have limited and read-only access.
```
Comment (frenck) [inline source/_components/device_tracker.fortios.markdown]:
```suggestion
    description: See [Fortinet Developer Network](https://fndn.fortinet.com) for how to create an API token. Remember this component only needs read access to a FortiGate, so configure the API user to only to have limited and read-only access.
```

File: source/_components/device_tracker.fortios.markdown [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+    required: true
+    type: string
+  verify_ssl:
+    description: if the SSL certificate should be verified. In most home cases users do not have a verified certificate. Default=False.
+    required: false
+    type: boolean
+{% endconfiguration %}
```
Comment (frenck) [inline source/_components/device_tracker.fortios.markdown]:
```suggestion
{% endconfiguration %}

```

**Review by frenck — CHANGES_REQUESTED** [review]
@kimfrellsen I've reviewed your PR and left some comments. Could you please take a look? Thanks! 👍

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
Please rename the file from `device_tracker.fortios.markdown` to `fortios.markdown`

**Review by frenck — DISMISSED** [review]

File: source/_components/fortios.markdown [hunk 10]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Device Tracker FortiOS"
+description: "Instructions on how to use Fortinet FortiOS to track devices in Home Assistant."
+logo: fortinet.png
+ha_category:
+  - Presence Detection
+ha_release: 0.96
```
Comment (MartinHjelmare) [inline source/_components/fortios.markdown]: 0.97

File: source/_components/fortios.markdown [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+---
```
Comment (MartinHjelmare) [inline source/_components/fortios.markdown:9]: Please add `ha_iot_class`.

File: source/_components/fortios.markdown (hunk 10 above)
Comment (frenck) [reply source/_components/fortios.markdown]: Done

File: source/_components/fortios.markdown (hunk 11 above)
Comment (frenck) [reply source/_components/fortios.markdown:9]: Done

**Review by frenck — DISMISSED** [review]

**Review by MartinHjelmare — DISMISSED** [review]

**Comment by MartinHjelmare** [comment]
Sorry for messing up the project status. 🙈

**Comment by frenck** [comment]
@MartinHjelmare Don't worry about that! You doing great

**Review by frenck — APPROVED** [review]

**Pull request merged**